crates/fraud_proofs/global_merkle_root/storage/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+authors = { workspace = true }
+categories = ["cryptography::cryptocurrencies"]
+description = "Storage types and primitives used by Fuel core protocol."
```

Comment:
It would be nice to update crate related stuff, because I just copied `fuel-core-storage`=D

Reply:
Good point, cleaned it up in https://github.com/FuelLabs/fuel-core/pull/2553/commits/02ee8224cba69791fbfab1c58aeaae65c575e41d